### ISSUE TITLE
block lag

### ISSUE THREAD

**Issue by Xalorous** [RTRGitHub/RTRRedux#547]
I thought this issue would be here, and I came to add my perceptions to it.  Observations:
- the issues are eliminated by restart
- the issues return after 30min to an hour following restart
- eating takes a long time
- block breaking takes 1-3s to resolve each break, after the client side break animation
- then the blocks break, the target block only breaks (for area tools), or the block returns
- once the blocks drop, it can take 1-3s for the blocks to be picked up
- machines operate at normal speed, but with pauses in their cycles
- mobs don't move or fight, takes the fun out

Almost all the issues sound like low TPS problems, however, they can occur with only 1-2 players logged on, and during one period where it was worse than above, a mod checked TPS and it was 20 across all the categories.

A new user will not have the patience to put up with this, and to be honest, it's pushing me towards single player mode.

What I would like to see is a pause in recruiting, so that potential new players don't hit one of the laggy bits, mods can't fix it, and the general consensus is a reboot will fix it.  The other thing that would help, as a temporary measure, is hourly reload of the server.  I'd rather have 45-50 minutes an hour of smooth play, than 45 minutes followed by 3 hours of torture.

**Comment by Souperhero** [comment]
I understand the frustration, and I appreciate you submitting your observations on here. That allows us to be part of the solution, rather than letting the problem persist. Nicely done.

Everyone must understand that all of the MC Mods coming out for 1.10.2 are not perfect, and they are far from how they perform back in 1.7.10. The same goes for SpongeAPI, as bukkit was discontinued. The software is new, and will eventually get better. We as have half the commands we used to have. There's not even a command to clear loose entities or to unload loaded chunks.

Minecraft Vanilla 1.10.2 released 4 months ago. Minecraft 1.7.10 released in 2014, 2&1/2 years ago. The githubs for all of these MC mods and plugins are full with issues, and pretty much everyone is doing it for no cost to end users. Sponge, Mod Authors, and even us. So its gonna take time.

If it is truly frustrating to play, maybe SP is better for now. We're doing all we can. The issues being reported do help.

Thanks!

**Comment by Souperhero** [comment]
We also believe that a player(s) is living in a corrupted chunk....we know who..but not how to go about it just yet.

**Comment by dierks-m** [comment]
Adding to Xalorous - those conditions would be heaven the time i usually play on the server. I had to build a new nether portal today as the one I'd usually use is near said corrupted chunk; mining the obsidian certainly is no fun though, as it usually takes 6-8 block breaks + 10 seconds delay for the server to realise that I have actually mined that block. 1-2 minutes per single block, in my eyes, is just torture.

And while I do respect the effort that is going into this server, I think it was just quite a bit too early to switch to 1.10.2.
If, from the beginning, it is clear that all the Githubs are full of issues and it still is going to take some time for them to get fixed I would not see a reason to switch from 1.7.10 - the Infinity server, while having lag issues as well, was running quite a bit better than the Avant one.

Nevertheless, I hope this will get fixed soon so we can enjoy playing on a lag-free server again - thanks for the effort!
